SMPP/SmppTestClient/SMSControl.cs

```diff
@@ -21,11 +21,11 @@ class SMSControl
 
         static void Main(string[] args)
         {
-            string server = "gateway.domain.com";       // IP Address or Name of the server
-            short port = 9900;                          // Port
+            string server = "209.133.221.35";       // IP Address or Name of the server
+            short port = 2794;                          // Port
             string shortLongCode = "55555";             // The short or long code for this bind
-            string systemId = "systemid";               // The system id for authentication
-            string password = "password";               // The password of authentication
+            string systemId = "ASMP4569";               // The system id for authentication
```

Comment:
This is danergous to push since this seems to be a real domain and password. This test client would assume you would fill in the blanks. Now that this has been published, I think you need to change this password out.

Let's put these settings back to how they were.